packages/python-peid/PKGBUILD

```diff
@@ -0,0 +1,34 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-peid
+_pyname=peid
+pkgver=1.2.9
+pkgrel=1
+pkgdesc='Python implementation of the Packed Executable iDentifier (PEiD)'
+arch=('any')
+grouos=('blackarch' 'blackarch-binary' 'blackarch-reversing')
+license=('GPL3')
+depends=('python' 'python-pefile' 'python-pip')
+makedepends=('python-setuptools')
+source=("$pkgname::git+https://github.com/packing-box/peid")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  cat src/peid/VERSION.txt
+}
+
+build() {
+  cd $pkgname
+
+  pip wheel .
+}
+
+package() {
+  cd $pkgname
+  pip install "./$_pyname-$pkgver-py3-none-$arch.whl" --root $pkgdir
+  install -Dm 644 LICENSE -t "${pkgdir}/usr/share/licenses/${pkgname}"
```

Comment:
Only install license if required according to  https://wiki.archlinux.org/index.php/PKGBUILD#license